apiserver/facades/client/modelmanager/services.go

```diff
@@ -228,9 +250,12 @@ type CredentialService interface {
 // AccessService defines a interface for interacting the users and permissions
 // of a controller.
 type AccessService interface {
-	// GetUserByName returns a User for the given name.
-	GetUserByName(context.Context, coreuser.Name) (coreuser.User, error)
-
+	// The following errors can be expected:
+	// - [github.com/juju/juju/domain/access/errors.UserNotFound] when no user
+	// exists for the supplied user name.
+	// - [github.com/juju/juju/domain/access/errors.UserNameNotValid] when the
+	// user name is not valid.
+	GetUserUUIDByName(context.Context, coreuser.Name) (coreuser.UUID, error)
```

Comment:
We can avoid adding this and just hard code the user to the "admin" user since model owners are being replaced by namespace.

Reply:
This is fundamental to a lot of logic at the moment. The proposal PR can refactor and provide a look at what this might look like. Doing this change insolation doesn't seem correct.